launch_xml/README.md

```diff
@@ -0,0 +1,67 @@
+# launch_xml
+
+This package provides an abstraction of the XML tree.
+
+## XML front-end mapping rules
```

Comment:
@ivanpauno we should also document the attribute lookup behavior (first check attributes in tags, then check attributes as children, then fail).